homeassistant/components/plex/server.py

```diff
@@ -219,6 +254,11 @@ def url_in_use(self):
         """Return URL used for connected Plex server."""
         return self._plex_server._baseurl  # pylint: disable=protected-access
 
+    @property
+    def ignore_new_shared_users(self):
```

Comment:
I recommend prepending with options_ to properties referring to config entry options

Reply:
Good suggestion, done!